detection/run_luna16_offical_eval.sh

```diff
@@ -25,7 +25,7 @@ python ./luna16_post_combine_cross_fold_results.py \
 	-o ./result/result_luna16_all.csv
 
 mkdir -p ./result/eval_luna16_scores
-python2 ./evaluation_luna16/noduleCADEvaluationLUNA16.py \
+python ./evaluation_luna16/noduleCADEvaluationLUNA16.py \
```

Comment:
Maybe use `python3` here to distinguish it from the previous version?

Reply:
All the MONAI programs use python3 as default.

Thanks.